### ISSUE TITLE
Video-4: Setup Our Tools

### ISSUE THREAD

**Issue by jomazu** [AlohaCode/mongodb-simple-complete#1]
When running `mongod --bind_ip=$IP --nojournal` just after `npm install`, I received an error stating `mongod: command not found`?  

I checked the `package.json` file and `"mongodb": "^2.1.7"` was listed under `"dependencies":`?

Nevertheless, I obtained the following solution from [Cloud9](https://community.c9.io/t/solved-mongod-command-not-found/6517/3)'s support page, which allowed me to move forward without issue ...

1. `npm install`
2. **==>** `sudo apt-get install mongodb-org` **<==**
3. Then run `mongod --bind_ip=$IP --nojournal`

**Comment by AlohaCode** [comment]
Yes, the dependency in package.json is for the Mongo Client Library as used in the node application, not for installing the database in your dev environment.   Apparently, MongoDB was not installed by default in your C9 environment.  The command "sudo apt-get install mongodb-org" installs the database in your environment so you can configure it, fire it up, and make db calls. 
Hope you were able to sort it all out!
